Source/driver/Castro.cpp

```diff
@@ -848,6 +851,11 @@ Castro::initData ()
 {
     BL_PROFILE("Castro::initData()");
 
+    // initialize the C++ values of the runtime parameters
+#ifdef PROB_PARAMS
+    init_prob_parameters();
```

Comment:
`initData()` is called on each level, and is not called after a restart -- can this be put in `variableSetup()` instead?

Reply:
yes, thanks.  I wasn't sure the best place to put it.

Reply:
okay, I moved it to the Castro constructor  and verified that the data is properly initialized